### PR TITLE
Implement docker-compose base 

### PR DESCRIPTION
**Pull request by Lewiky**
Configure docker-compose to orchestrate all services from this point on.

Also includes base implementation of BrendonService.

### CODE REVIEW

File: BrendonService/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM tiangolo/uwsgi-nginx-flask:python3.6-alpine3.7
+RUN apk --update add bash vim
```
Comment (jaylees14) [inline BrendonService/Dockerfile]: Why vim?

File: BrendonService/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+COPY ./requirements.txt /var/www/requirements.txt
+RUN pip install -r /var/www/requirements.txt
+EXPOSE 56733
```
Comment (xsanda) [inline BrendonService/Dockerfile]: mAgIc NuMbErS

File: BrendonService/main.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from app import app
```
Comment (jaylees14) [inline BrendonService/main.py]: Maybe rename to flask_app or something?

Comment (xsanda) [reply BrendonService/main.py]: And rename the package to BrendonService?

**Pull request merged**